.github/workflows/cli-tests.yaml

```diff
@@ -64,12 +64,23 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version-file: ./go.mod
-      - run:  go run ./cmd/devbox run fmt
-      - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3.7.0
+
+      - name: Install devbox
+        uses: jetpack-io/devbox-install-action@v0.7.0
+        with:
+         enable-cache: true
+
+      - name: Mount golang cache
+        uses: actions/cache@v3
         with:
-          args: "--out-${NO_FUTURE}format colored-line-number --timeout=10m"
-          skip-cache: true
+          path: |
+            ~/.cache/golangci-lint
+            ~/.cache/go-build
+            ~/go/pkg
+          key: ${{ runner.os }}-${{ hashFiles('go.work.sum') }}
```

Comment:
suggestion:
```
key: ${{ runner.os }}-devbox-go-${{ hashFiles('go.work.sum') }}
restore-keys: ${{ runner.os }}-devbox-go-
```

https://github.com/actions/cache/blob/main/caching-strategies.md#using-restore-keys-to-download-the-closest-matching-cache

This may make it faster in the event of a cache miss, since it'll use the pre-existing "closest" cache as a starting point and then download the handful of changed dependencies to update the cache.

Reply:
Agree with using restore keys, but not with adding `devbox-go-`. (I would just use a generic `go` or `cache-go` like the example you shared). This would allow sharing between go projects if they happen to have identical dependencies. (though that is rare)